src/util.rs

```diff
@@ -44,11 +44,11 @@ pub fn print_result(r: Response, resource_name: String, op: Verbs) {
 
 
 pub fn url_validation(url: Option<&str>) -> Result<Url> {
-    Url::parse(url.unwrap()).context(format!("URL args: \'{}\' is not valid", url.unwrap()))
+    Url::parse(url.unwrap()).context(format!("URL args: \'{}\' is not valid", url.unwrap_or("")))
```

Comment:
The function already returns a result, the unwrap can be replaced with `?` if i am correct